tests/data/deepglobelandcover/data.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+
+import numpy as np
+from PIL import Image
+from torchvision.datasets.utils import calculate_md5
+
+
+def generate_test_data(root: str, n_samples: int = 3) -> str:
+    """Creates test data archive for DeepGlobeLandCover dataset and
+    returns its md5 hash.
+    Args:
+        root (str): Path to store test data
+        n_samples (int, optional): Number of samples.
+    Returns:
+        str: md5 hash of created archive
+    """
+    dtype = np.uint8
+    size = 2
+
+    folder_path = os.path.join(root, "data")
+
+    train_img_dir = os.path.join(folder_path, "data", "training_data", "images")
+    train_mask_dir = os.path.join(folder_path, "data", "training_data", "masks")
+    test_img_dir = os.path.join(folder_path, "data", "test_data", "images")
+    test_mask_dir = os.path.join(folder_path, "data", "test_data", "masks")
+
+    if not os.path.exists(train_img_dir):
+        os.makedirs(train_img_dir)
```

Comment:
```suggestion
    os.makedirs(train_img_dir, exist_ok=True)
```
Can do this for other lines too